# Periodic state update of gripper

bool is_gripped # True: is gripping, False: open

float32 left_pos
float32 right_pos

float32 load_left
float32 load_right
float32 avg_load

float32 avg_temp